doc/reference/kernel/threads/workqueue.rst

```diff
@@ -122,6 +122,41 @@ queue, unless the work item has already been removed and processed by the
 workqueue's thread. Consequently, once a work item's timeout has expired
 the work item is always processed by the workqueue and cannot be canceled.
 
+Triggered Work
+**************
+
+The :cpp:func:`k_work_poll()` interface schedules a triggered work
+item in response to a **poll event** (see :ref:`polling_v2`), that will
+call a user-defined function when the monitored resource becomes available
+or a poll signal is raised. In contrast to :cpp:func:`k_poll()`,
```

Comment:
" ~the~ _a_ monitored resource becomes available or ~a~ poll signal is raised, _or a timeout occurs_."

Reply:
Fixed